test.sh

```diff
@@ -41,6 +41,8 @@ for d in $PKG; do
 done
 
 go vet -all ./...
-golangci-lint run -D errcheck ./...  # TODO: Enable errcheck back.
+if [ `go version | { read _ _ v _; echo ${v#go}; }` != "devel" ];  then
```

Comment:
Could we rather do this similar to how we handle the staticcheck? We also skip it on tip, via RUN_STATICCHECK env var. See above and ci.yaml (note that it's set in two places there).

Reply:
Done.